testsuite/features/support/env.rb

```diff
@@ -136,8 +136,8 @@ def capybara_register_driver
       path = "screenshots/#{scenario.name.tr(' ./', '_')}.png"
       # only click on Details when we have errors during bootstrapping and more Details available
       click_button('Details') if has_content?('Bootstrap Minions') && has_content?('Details')
-      # a TimeoutError may be raised while a page is still (re)loading
-      find('#page-body', wait: 3) if scenario.exception.is_a?(TimeoutError)
+      # a Timeout::Error may be raised while a page is still (re)loading
+      find('#page-body', wait: 3) if scenario.exception.is_a?(Timeout::Error)
```

Comment:
Note: I think is a good practice to `require` the dependency we use on every Ruby file using it. So, imho, it would be better to also include in env.rb, this line `require 'timeout'`